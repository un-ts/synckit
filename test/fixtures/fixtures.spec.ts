import path from 'node:path'

import { exec } from 'tinyexec'

test('yarn-pnp', async () => {
  const result = await exec('yarn', ['node', 'index.js'], {
    nodeOptions: {
      cwd: path.resolve('test/fixtures/yarn-pnp'),
      env: {
        FORCE_COLOR: '0',
      },
    },
  })

  expect(result).toMatchSnapshot()
})

test('bun', async () => {
  const result = await exec('bun', ['index.ts'], {
    nodeOptions: {
      cwd: path.resolve('test/fixtures/bun'),
      env: {
        FORCE_COLOR: '0',
      },
    },
  })

  expect(result).toMatchSnapshot()
})
