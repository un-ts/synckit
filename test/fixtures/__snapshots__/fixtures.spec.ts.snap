// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bun 1`] = `
{
  "exitCode": 0,
  "stderr": "",
  "stdout": "Worker: 1
Worker: 2
Worker: 5
1 2 5
",
}
`;

exports[`yarn-pnp 1`] = `
{
  "exitCode": 0,
  "stderr": "",
  "stdout": "Worker: 1
Worker: 2
Worker: 5
1 2 5
",
}
`;
